pkg/app/var_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestGetFieldAsString(t *testing.T) {
+	m := map[string]interface{}{
+		"Kind": "Service",
+		"metadata": map[string]interface{}{
+			"labels": map[string]string{
+				"app": "application-name",
+			},
+			"name": "service-name",
+		},
+	}
+	p := []string{"Kind"}
+	s, _ := getFieldAsString(m, p)
+	if s != "Service" {
+		t.Errorf("Expected to get Service, but actually got %s", s)
+	}
```

Comment:
please format as a tabular test, e.g. pkg/resmap/kv_test.go

Reply:
this is done.